src/index.js

```diff
@@ -43,12 +43,24 @@ fetch.mockResponse = (body, init) => {
   );
 };
 
+fetch.mockReject = (body, init) => {
```

Comment:
Do you need to pass body, init into `mockReject` when they are not used? (same for `MockRejectOnce`)